modules/migration-migrating-applications-api.adoc

```diff
@@ -0,0 +1,367 @@
+    awsBucketName: <bucket_name> <1>
+    credsSecretRef:
+      name: <storage_secret_ref> <2>
+      namespace: openshift-config
+  backupStorageProvider: <storage_provider_name> <3>
+  volumeSnapshotConfig:
+    credsSecretRef:
+      name: <storage_secret_ref> <4>
+      namespace: openshift-config
+  volumeSnapshotProvider: <storage_provider_name> <5>
+----
+<1> Specify the bucket name.
+<2> Specify the `Secrets` CR of the object storage. You must ensure that the credentials stored in the `Secrets` CR of the object storage are correct.
+<3> Specify the storage provider.
+<4> Optional: If you are copying data by using snapshots, specify the `Secrets` CR of the object storage. You must ensure that the credentials stored in the `Secrets` CR of the object storage are correct.
+<5> Optional: If you are copying data by using snapshots, specify the storage provider.
+
+. Create the `MigStorage` CR:
++
+[source,terminal]
+----
+$ oc create -f migstorage.yaml -n openshift-migration
+----
+
+. Verify that the `MigStorage` CR is in a `Ready` state:
++
+[source,terminal]
+----
+$ oc describe migstorage <migstorage_name>
+----
+
+. Create a `MigPlan` CR manifest called `migplan.yaml`:
++
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigPlan
+metadata:
+  name: <migration_plan>
+  namespace: openshift-migration
+spec:
+  destMigClusterRef:
+    name: host
+    namespace: openshift-migration
+  indirectImageMigration: true <1>
+  indirectVolumeMigration: true <2>
+  migStorageRef:
+    name: <migstorage_ref> <3>
+    namespace: openshift-migration
+  namespaces:
+    - <application_namespace> <4>
+  srcMigClusterRef:
+    name: <remote_cluster_ref> <5>
+    namespace: openshift-migration
+----
+<1> Direct image migration is enabled if `false`.
+<2> Direct volume migration is enabled if `false`.
+<3> Specify the name of the `MigStorage` CR instance.
+<4> Specify one or more namespaces to be migrated.
+<5> Specify the name of the `source` cluster `MigCluster` instance.
+
+. Create the `MigPlan` CR:
++
+[source,terminal]
+----
+$ oc create -f migplan.yaml -n openshift-migration
+----
+
+. View the `MigPlan` instance to verify that it is in a `Ready` state:
++
+[source,terminal]
+----
+$ oc describe migplan <migplan_name> -n openshift-migration
+----
+
+. Create a `MigMigration` CR manifest called `migmigration.yaml`:
++
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigMigration
+metadata:
+  name: <migmigration_name>
+  namespace: openshift-migration
+spec:
+  migPlanRef:
+    name: <migplan_name> <1>
+    namespace: openshift-migration
+  quiescePods: true <2>
+  stage: false <3>
+  rollback: false <4>
+----
+<1> Specify the `MigPlan` CR name.
+<2> The pods on the `source` cluster are stopped before migration if `true`.
+<3> A stage migration, without stopping the application, is performed if `true`. A full migration, which stops the application and migrates the PVs, is performed if `false`.
```

Comment:
the description is not accurate regarding `A full migration`, A full migration migrates all things inside the selected namespace, including PVs, services, routes, deployments, etc

Reply:
@XinRedhat What term are we using for a migration that is not "stage"?

Reply:
@apinnick "migration", just reference the name on UI.